src/stylesheets/common/_colors.scss

```diff
@@ -57,46 +57,75 @@ $okColor:#63a240;
 
 
 //Classes for styleguide
```

Comment:
Or, are these classes only intended to be used on the common/styleguide/ pages?  In which case, you can pretty much ignore the above. Though we should probably move them to the styleguide-specific scss file.